vcap/vcap/backend.py

```diff
@@ -54,5 +75,7 @@ def batch_predict(self, input_data_list: List[Any]) -> List[Any]:
 
     @abc.abstractmethod
     def close(self) -> None:
-        """De-initializes the backend."""
+        """De-initializes the backend. Once this method is called, the backend
```

Comment:
I think this could be worded a bit better. It's a bit unclear if this function is called before/after/as the backend is no longer in use. `Once` is probably too ambiguous here

Reply:
Did it!